src/lib/components/ui/Tabs/__tests__/Tabs.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Tabs } from '../Tabs';
+
+describe('rendering', () => {
+  it('renders correctly without children', () => {
+    const tree = shallow(
+      <Tabs />,
+    );
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders correctly with children', () => {
+    const tree = shallow(
+      <Tabs id="my-tabs">
+        <li>item 1</li>
```

Comment:
We expect to have strict prop type definition later. We want to have `TabItem` here as `li` should never be passed to Tabs.

Reply:
Fixed.